code/stable/dblpendulum/SRS/DblPendulum_SRS.tex

```diff
@@ -761,7 +761,7 @@ \subsubsection{General Definitions}
         
 \\ \midrule \\
 Equation & \begin{displaymath}
-           T=2 π \sqrt{\frac{{L_{\text{rod}}}}{\mathbf{g}}}
+           T=2.0 π \sqrt{\frac{{L_{\text{rod}}}}{\mathbf{g}}}
```

Comment:
And this is where we need `Exact`. Even though in code, we'd want `2.0`, in the display math, we'll still want `2`. So we do need to embed the integers into the reals in our representation.